Cargo.toml

```diff
@@ -27,7 +27,7 @@ readme = "README.md"
 
 [workspace.dependencies]
 miden-air = { version = "0.9", default-features = false }
-miden-lib = { version = "0.4" }
+miden-lib = { rev = "5cb5047", git = "https://github.com/0xPolygonMiden/miden-base" }
```

Comment:
Thanks - I originally had it that way but then changed my mind since its still fairly early (?) in the cycle and its annoying having the foundation move.